meson.build

```
@@ -82,7 +82,7 @@ with_programs = get_option('programs')
 with_soup2 = get_option('soup2').enabled()
 if get_option('soup2').auto()
     # Prefer using API 1.1 level if available.
```

You've missed the comment.
```suggestion
    # Prefer using API 2.0 level if available.
```